shared/attributes.asciidoc

```diff
@@ -406,6 +406,7 @@ Issue and pull request URLs
 :ml-issue:     {ml-repo}issues/
 :ml-pull:      {ml-repo}pull/
 :ml-commit:    {ml-repo}commit/
+:java-cl-rep:  https://github.com/elastic/elasticsearch-java
```

Comment:
Should this be something like `:java-hlrc-repo:`. I wonder if `cl` is too vague?

Reply:
```suggestion
:java-hlrc-rep:  https://github.com/elastic/elasticsearch-java
```